scripts/toolboxSrc/applyEnvVarsToClientAssets.ts

```diff
@@ -98,3 +98,5 @@ const rewriteIndexHTML = async () => {
 export const applyEnvVarsToClientAssets = async () => {
   return Promise.all([rewriteServiceWorker(), rewriteIndexHTML(), writeManifest()])
 }
+
+if (require.main === module) applyEnvVarsToClientAssets()
```

Comment:
writing the 3 client assets takes about 20ms, so even though it's a bit wasteful to do it every time we call the script, it's simple & grants the opportunity to change env vars easily without storing a checksum somewhere